dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/User32.cs

```diff
@@ -45,9 +69,8 @@ public enum WM
            IntPtr hInstance,
            IntPtr lpParam);
 
-        [DllImport("user32", SetLastError = true)]
+        [DllImport("user32.dll", EntryPoint = "DestroyWindow", CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
Use "user32" not "user32.dll". Either will work, but for consistency we should do it the same way each time.